### PR TITLE
AMPI Benchmarks: Make pingpong run on a range of message sizes

### PR DESCRIPTION
**Pull request by nitbhat**

### CODE REVIEW

File: benchmarks/ampi/pingpong/pingpong-1way.c [hunk 1]
```diff
@@ -1,80 +1,116 @@
...
+
   MPI_Comm_rank( MPI_COMM_WORLD, &my_id );
   MPI_Comm_size( MPI_COMM_WORLD, &p );
-  
-  if ((sscanf (argv[1], "%d", &max_msgs) < 1) ||
-      (sscanf (argv[2], "%d", &msg_size) < 1)) {
-    fprintf (stderr, "need msg count and msg size as params\n");
-    goto EXIT;
+
+  printf("[Rank: %d][Num procs: %d] argc = %d\n", my_id, p, argc);
```
Comment (stwhite91) [inline benchmarks/ampi/pingpong/pingpong-1way.c]: This should be removed

Comment (nitbhat) [reply benchmarks/ampi/pingpong/pingpong-1way.c]: Done.

File: benchmarks/ampi/pingpong/pingpong-1way.c [hunk 2]
```diff
@@ -1,80 +1,114 @@
...
+    else
+      iter = high_iter;
+    run(msg_size, iter, my_id, p, message_s, message_r, printFormat);
+    msg_size *= 2;
+  }
+
+  free(message_s);
+  free(message_r);
+
+  return finish();
```
Comment (matthiasdiener) [inline benchmarks/ampi/pingpong/pingpong-1way.c]: This seems to return unsuccessful execution (-1) even when it was successful?

File: benchmarks/ampi/pingpong/pingpong-1way.c [hunk 3]
```diff
@@ -1,80 +1,114 @@
...
+    min_msg_size = atoi(argv[1]);
+    max_msg_size  = atoi(argv[2]);
+    low_iter = atoi(argv[3]);
+    high_iter = atoi(argv[4]);
+    printFormat = atoi(argv[5]);
   }
 
-  message_s = (char*)malloc (msg_size);  
-  message_r = (char*)malloc (msg_size);
+  message_s = (char*)malloc (max_msg_size);
```
Comment (matthiasdiener) [inline benchmarks/ampi/pingpong/pingpong-1way.c]: The actual data that gets send is actually undefined? Maybe add a comment here or change the malloc to calloc?

Comment (nitbhat) [reply benchmarks/ampi/pingpong/pingpong-1way.c]: Modified the allocation to `calloc`.

File: benchmarks/ampi/pingpong/pingpong-1way.c (hunk 2 above)
Comment (nitbhat) [reply benchmarks/ampi/pingpong/pingpong-1way.c]: Removed the `finish` method and added code for successful return.

**Pull request merged**